src/js/tour.js

```diff
@@ -328,6 +332,13 @@ export class Tour extends Evented {
       this._hideModalOverlay();
 
       document.body.appendChild(this._modalOverlayElem);
+
+      const overlay = document.querySelector('#shepherdModalOverlayContainer');
+      // Prevents window from moving on touch.
+      window.addEventListener('touchmove', preventModalBodyTouch, false);
+
+      // Allows content to move on touch.
+      overlay.addEventListener('touchmove', preventModalOverlayTouch, false);
```

Comment:
Similar question: Perhaps we could move this to the `addStepEventListeners`  util.